### PR TITLE
Copy edited the release notes.

### PR DESCRIPTION
**Pull request by paperdigits**
Copy edited the release notes.

### CODE REVIEW

File: RELEASE_NOTES.md [hunk 1]
```diff
@@ -28,16 +28,7 @@ making a backup is strongly advised.
...
 ## The Big Ones
 
-- A full rewrite of the lighttable has been undertaken. This gives again a
-  big performance improvement and will allow smooth interaction at any
-  screen resolution. At the same time the filmstrip has been reworked.
-
-  It is important to note that this speed improvement is really
-  noticeable, particularly in the zoomable lighttable view.
-
-  The culling view has also been rewritten from scratch.
```
Comment (TurboGit) [inline RELEASE_NOTES.md:38]: I'm ok with all changes except this. I think we should keep the culling view and 8k notes. Can you propose an update with to this PR? TIA.

Comment (paperdigits) [reply RELEASE_NOTES.md:38]: @TurboGit yes, done and pushed.

File: RELEASE_NOTES.md [hunk 2]
```diff
@@ -116,14 +105,11 @@ making a backup is strongly advised.
...
-- Filmic RGB goes to v4 (new color science) with integrated highlight recovery.
+- Filmic RGB is updated to v4 (new color science) with integrated highlight recovery.
 
 ## New Features And Changes
 
-- Add support for curved gradients. The gradient mask can now be
-  curved to revert fish-eye kind distortion and so become a straight
-  line on the horizon for example. This can also be used to artistic
-  goals.
+- Add support for curved gradients. This can be helpful when putting a gradient mask on an  image with a horizon line that is curved because lens distortion. This can also be used towards artistic goals.
```
Comment (elstoc) [inline RELEASE_NOTES.md:112]: "...curved **due to** lens distortion..."
"...can also be used **for** artistic goals."

File: RELEASE_NOTES.md [hunk 3]
```diff
@@ -201,8 +187,8 @@ making a backup is strongly advised.
   having created them. This is achieved by dragging one of the four
   little square handles at the corner.
 
-- Tagging improvements: Entry tag(s) creation works now without image
-  selected.  It is allowed to create a tag on virtual node, to insert
+- Tagging improvements: Entry tag(s) creation works now without an image
+  selected.  It is allowed the user to create a tag on a virtual node, to insert
   a pipe <kbd>|</kbd> character in create tag (menu). The tree display
   shows the newly created tags.
```
Comment (elstoc) [inline RELEASE_NOTES.md:194]: I'm not sure what the second sentence of this paragraph means

File: RELEASE_NOTES.md [hunk 4]
```diff
@@ -234,7 +220,7 @@ making a backup is strongly advised.
 
 - Add some tooltip information for tone equalizer.
 
-- Some actions like cropping are more responsive by triggering a
+- Some actions, like cropping, are more responsive by triggering a
```
Comment (elstoc) [inline RELEASE_NOTES.md:223]: "...**have been made** more responsive..."

File: RELEASE_NOTES.md [hunk 5]
```diff
@@ -280,7 +266,7 @@ making a backup is strongly advised.
 - Optimize the denoise profile module (bilateral filter) for better
   performance.
 
-- Many part of the histogram code has been reworked for better
+- Many parts of the histogram code has been reworked for better
```
Comment (elstoc) [inline RELEASE_NOTES.md:269]: "...**have** been reworked for better..."

**Comment by TurboGit** [comment]
Thanks !

File: RELEASE_NOTES.md [hunk 6]
```diff
@@ -350,8 +336,7 @@ making a backup is strongly advised.
 
 - Fix gphoto camera detection procedure.
 
-- Fix the opacity (a second attempt) issue which could go wrong and
-  let thinking that a mask had no effect.
+- Fix the opacity (a second attempt) issue which lead to a mask having no effect.
```
Comment (elstoc) [inline RELEASE_NOTES.md:339]: "Fix the opacity issue (second attempt) which led to a mask having no effect."

**Comment by elstoc** [comment]
d'oh too late!

**Pull request merged**